src/asciidoc/docs/develop/editor.adoc

```diff
@@ -78,3 +80,51 @@ You can scan this code to preview the template on your mobile device.
 === Launching the debugger
 
 *Launch Debugger* opens the debugger in a new browser tab.
+
+
+[.faq]
+== Tips
+[id="faq-nodejsnpmver"]
+.What versions of Node.js and npm do Kojis run on?
+[%collapsible]
+====
+include::{includespath}/misc.adoc[tag=nodejsnpmver]
+====
+
+[id="faq-revert-kojiversion"]
+.How do I revert to an old version of my template?
+[%collapsible]
+====
+To revert to a previous commit,
```

Comment:
```suggestion
To revert your source code to a previous commit:
```
Added some context to clarify this lead-in. Also, used a colon to match other procedures.

Reply:
Fixed. Will be in the next commit.